mixcoatl/resource.py

```diff
@@ -52,7 +52,19 @@ class Resource(object):
     #: The unique identifier of an individual resource
     PRIMARY_KEY = None
 
-    def __init__(self, base_path=None, request_details='basic', **kwargs):
+    def __init__(self,
+                 base_path=None,
+                 request_details='basic',
+                 config=None,
+                 **kwargs):
+
+        if config:
+            self.config = config
+        else:
+            # use the default config.
+            self.config = Config()
```

Comment:
I would prefer to see this as:
self.config = settings

This would be more inline with the previous behavior.